website/docs/r/virtual_desktop_workspace.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+subcategory: "Desktop Virtualization"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_virtual_desktop_workspace"
+description: |-
+  Manages a Virtual Desktop Workspace.
+---
+
+# azurerm_virtual_desktop_workspace
+
+Manages a Virtual Desktop Workspace.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "rg-example-virtualdesktop"
+  location = "eastus"
+}
+
+resource "azurerm_virtual_desktop_workspace" "workspace" {
+  name                = "workspace"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  friendly_name = "FriendlyName"
+  description   = "A description of my workspace"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Virtual Desktop Workspace. Changing the name
+    forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to
+    create the Virtual Desktop Workspace. Changing the resource group name forces
+    a new resource to be created.
+
+* `location` - (Required) The location/region where the Virtual Desktop Workspace is
+    located. Changing the location/region forces a new resource to be created.
```

Comment:
would we remove all line breaks from property descriptions in the docs? this is the old way we wrote them
```suggestion
* `location` - (Required) The location/region where the Virtual Desktop Workspace is located. Changing the location/region forces a new resource to be created.
```